airflow/utils/log/task_context_logger.py

```diff
@@ -78,15 +78,20 @@ def _get_task_handler() -> FileTaskHandler | None:
             assert isinstance(h, FileTaskHandler)
         return h
 
-    def _log(self, level: int, msg: str, *args, ti: TaskInstance):
+    def _log(self, level: int, msg: str, *args, ti: TaskInstance, suppress_in_call_site=False):
```

Comment:
I think this should be more descriptive. Maybe something like `task_instance_logs_only`?